### PR TITLE
Add a blog post about the permissions page

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

File: _posts/2018-03-19-permissions.markdown [hunk 1]
```diff
@@ -5,12 +5,19 @@ date:   2018-03-19 18:34:49 +0000
 categories: jsdrupal planet
 ---
 
-As of today we've merged the first iteration of a better permissions page:
+This blogpost is a small incremental success we had in the JS initiative.
+Last week we merged the first iteration of a permissions page.
+Let's see how it looks like:
```
Comment (mattgrill) [inline _posts/2018-03-19-permissions.markdown]: `Let's see how it looks like:` -> `Let's see what it looks like:`

File: _posts/2018-03-19-permissions.markdown [hunk 2]
```diff
@@ -5,12 +5,19 @@ date:   2018-03-19 18:34:49 +0000
...
-![A better permissions UI](https://user-images.githubusercontent.com/1090713/37614021-c8b23380-2b66-11e8-80bc-fc791835d0d8.gif)
+![A gif of the new permission UI]({{ "/assets/permissions.gif" | absolute_url }})
 
-As you can see on this preview: We've added support to filtering permissions by name. This gives you an easy way to for example
-find all permissions related to users, or a specific content type.
+There are few things to point out:
+
+* For now it looks really similar to the permissions page in Drupal. We are looking forward to work together 
+with the [Usability team](https://groups.drupal.org/usability)
+* A tiny little feature we managed to implement: Dynamic filtering as you type. We believe these small UX improvements can make a huge difference, as you can quickly filter for example by everything related to the "Page" content type.
```
Comment (mattgrill) [inline _posts/2018-03-19-permissions.markdown]: I'd perhaps not belittle the work we've done so far? Maybe remove `tiny little`.

File: _posts/2018-03-19-permissions.markdown [hunk 3]
```diff
@@ -5,12 +5,19 @@ date:   2018-03-19 18:34:49 +0000
...
+
+As you could imagine improving the user experience of Drupal is a huge undertaking. The technical side is
```
Comment (mattgrill) [inline _posts/2018-03-19-permissions.markdown]: `could` -> `can`

**Review by mattgrill — CHANGES_REQUESTED** [review]

File: _posts/2018-03-19-permissions.markdown (hunk 2 above)
Comment (dawehner) [reply _posts/2018-03-19-permissions.markdown]: +1

**Review by mattgrill — APPROVED** [review]

**Comment by mattgrill** [comment]
:shipit:

File: _posts/2018-03-19-permissions.markdown [hunk 4]
```diff
@@ -5,12 +5,19 @@ date:   2018-03-19 18:34:49 +0000
...
-As you can see on this preview: We've added support to filtering permissions by name. This gives you an easy way to for example
-find all permissions related to users, or a specific content type.
+There are few things to point out:
+
+* For now it looks really similar to the permissions page in Drupal. We are looking forward to work together 
+with the [Usability team](https://groups.drupal.org/usability)
+* A feature we managed to implement: Dynamic filtering as you type. We believe these small UX improvements can make a huge difference, as you can quickly filter for example by everything related to the "Page" content type.
+
+As you can imagine improving the user experience of Drupal is a huge undertaking. The technical side is
+actually by far the easiest bit. Coming up with good, simple but also flexible design pattterns is much harder. We need help, so please [get involved](https://jsdrupal.github.io/jsdrupal/get-involved/).
```
Comment (lauriii) [inline _posts/2018-03-19-permissions.markdown]: s/pattterns/patterns

Comment (lauriii) [inline _posts/2018-03-19-permissions.markdown]: Should that link lead to something else than 404?

**Review by lauriii — CHANGES_REQUESTED** [review]

File: _posts/2018-03-19-permissions.markdown (hunk 4 above)
Comment (dawehner) [reply _posts/2018-03-19-permissions.markdown]: :+1:

Comment (dawehner) [reply _posts/2018-03-19-permissions.markdown]: Fixed. This was the previous URL.

**Review by lauriii — APPROVED** [review]

**Comment by lauriii** [comment]
Looks good!

**Comment by dawehner** [comment]
Should we merge it?

**Pull request merged**